src/txn/txn.c

```diff
@@ -2801,7 +2806,8 @@ __wt_verbose_dump_txn_one(
         ", flags: 0x%08" PRIx32 ", isolation: %s"
         ", last saved error code: %d"
         ", last saved sub-level error code: %d"
-        ", last saved error message: %s",
+        ", last saved error message: %s"
+        ", prepared id: %" PRIu64,
```

Comment:
It would be good to display the `prepare_id` along with the `prepare_timestamp`, instead of at the end of the data.

Reply:
fixed